From 59100b60d2aeaa9110346995777b6ffc255e8999 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 8 Sep 2021 06:57:09 +0200
Subject: [PATCH 4/5] loosen requirements

* Set grpcio pin to the built grpc version
* Remove unused clang requirement
---
 tensorflow/tools/pip_package/setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tensorflow/tools/pip_package/setup.py b/tensorflow/tools/pip_package/setup.py
index 26306e43aaf..bf5d21aca5b 100644
--- a/tensorflow/tools/pip_package/setup.py
+++ b/tensorflow/tools/pip_package/setup.py
@@ -82,7 +82,6 @@ REQUIRED_PACKAGES = [
     # Install other dependencies
     'absl-py >= 0.4.0',
     'astunparse >= 1.6.0',
-    'libclang >= 9.0.1',
     'flatbuffers >= 1.12, < 3.0', # capped as jax 0.1.71 needs < 3.0
     'google_pasta >= 0.1.1',
     'h5py >= 2.9.0', # capped since 3.3.0 lacks py3.6
@@ -129,7 +128,7 @@ if 'tf_nightly' in project_name:
 # BoringSSL support.
 # See https://github.com/tensorflow/tensorflow/issues/17882.
 if sys.byteorder == 'little':
-  REQUIRED_PACKAGES.append('grpcio >= 1.24.3, < 2.0')
+  REQUIRED_PACKAGES.append('grpcio >= GRPCIO_VERSION, < 2.0')
 
 
 # Packages which are only needed for testing code.
-- 
2.32.0.windows.2

